### PR TITLE
Add tooltips to files in tree view.

### PR DESCRIPTION
**Pull request by mattgodbolt**
* Makes the two buttons (stage/unstage) separate so the template
  file has all the title etc in it.
* Hides/unhides to show the appropriate button only.

I'm not wed to this approach, but it saved putting the descriptions
into the source code.

### CODE REVIEW

File: static/panes/tree.ts [hunk 1]
```diff
@@ -22,9 +22,8 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import { MultifileFile, MultifileService, MultifileServiceState } from "../multifile-service";
-import { saveAs } from "file-saver";
-import { LineColouring } from "../line-colouring";
+import {MultifileFile, MultifileService, MultifileServiceState} from '../multifile-service';
```
Comment (mattgodbolt) [inline static/panes/tree.ts:25]: I don't remember changing this stuff...is it possible a prettier thing?

File: views/templates.pug [hunk 2]
```diff
@@ -478,6 +478,7 @@
   #tree-editor-tpl
     li.list-group-item.tree-editor-file.input-group-append
         span.filename someresource.txt
-        button.btn.delete-file.fa.fa-trash
-        button.btn.rename-file.fa.fa-tag
-        button.btn.stage-file.fa.fa-plus
+        button.btn.delete-file.fa.fa-trash(title="Remove this file" aria-label="Delete")
```
Comment (mattgodbolt) [inline views/templates.pug:481]: Comments welcome on the text here! I started playing with the UI to do a "howto" doc and thought 'ooh these should have tooltips!'

File: static/panes/tree.ts (hunk 1 above)
Comment (partouf) [reply static/panes/tree.ts:25]: must be

**Review by partouf — APPROVED** [review]
lgtm!

**Comment by mattgodbolt** [comment]
Thanks partouf!

File: static/panes/tree.ts (hunk 1 above)
Comment (RubenRBS) [reply static/panes/tree.ts:25]: Can't be. `requirePragma` is set in the prettier config and this file (As well as the rest of the files in the repo) lacks it

Comment (partouf) [reply static/panes/tree.ts:25]: hmm, weird, maybe matt's IDE did it?

Comment (partouf) [reply static/panes/tree.ts:25]: when I'm doing changes I don't see this either, so I guess it's true what you say @RubenRBS

**Comment by partouf** [comment]
this is now live

File: static/panes/tree.ts (hunk 1 above)
Comment (mattgodbolt) [reply static/panes/tree.ts:25]: Yes, very likely it was my IDE that did it (I hit reformat in the IDE which drops redundant includes) before I found the "format using Prettier" button.

**Pull request merged**